mne/stats/regression.py

```diff
@@ -387,3 +396,15 @@ def _make_evokeds(coefs, conds, cond_length, tmin_s, tmax_s, info):
             kind='average')  # nave and kind are technically incorrect
         cumul += tmax_ - tmin_
     return evokeds
+
+
+def _cho_solver(X, y):
+    a = (X.T * X).toarray()  # dot product of sparse matrices
+    return linalg.solve(a, X.T * y.T, sym_pos=True,
+                        overwrite_a=True, overwrite_b=True).T
```

Comment:
X.T is indeed a view (identical memory/data), but the results of multiplications will not be views, they will be new arrays